### PR TITLE
[MilkmApp] Integrate valid from decorator on create order

### PR DESCRIPTION
**Pull request by yvesfracari**
https://www.loom.com/share/671f4114708c433d943e5208928b849b

### CODE REVIEW

File: apps/cow-tools/src/lib/encode.ts [hunk 1]
```diff
@@ -6,6 +7,32 @@ import {
 } from "./priceCheckersMappings";
 import { argType, PRICE_CHECKERS, PriceCheckerArgument } from "./types";
 
+export function encodePriceCheckerDataWithValidFromDecorator(
+  priceCheckerAddress: Address,
+  priceCheckerData: `0x${string}`,
+  validFrom: string,
+) {
+  const validFromDate = new Date(validFrom);
+  const validFromTimestamp = BigInt(validFromDate.getTime() / 1000);
```
Comment (luizakp) [inline apps/cow-tools/src/lib/encode.ts:16]: we have a function that does that on `packages/utils/date.ts` called `dateToEpoch`

File: apps/cow-tools/src/lib/transactionFactory.ts
Comment (luizakp) [inline apps/cow-tools/src/lib/transactionFactory.ts:1]: 🔝

**Review by luizakp — DISMISSED** [review]
same as the other PR, just fix the build error and let's merge hehehe

File: apps/cow-tools/src/lib/priceCheckersMappings.ts [hunk 2]
```diff
@@ -142,6 +142,10 @@ export const priceCheckerAddressesMapping = {
   },
 } as const;
 
+export const validFromDecorator = {
```
Comment (ribeirojose) [inline apps/cow-tools/src/lib/priceCheckersMappings.ts:145]: what's this decorating? 👀

**Pull request merged**